src/main/java/frc/robot/subsystems/elevator/ElevatorMechanism.java

```diff
@@ -0,0 +1,314 @@
+package frc.robot.subsystems.elevator;
+
+import static edu.wpi.first.units.Units.Inches;
+import static edu.wpi.first.units.Units.Meters;
+import static edu.wpi.first.units.Units.RadiansPerSecond;
+import static edu.wpi.first.units.Units.Rotations;
+import static edu.wpi.first.units.Units.Volts;
+import static edu.wpi.first.units.Units.VoltsPerRadianPerSecond;
+import static edu.wpi.first.units.Units.VoltsPerRadianPerSecondSquared;
+
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.Distance;
+import edu.wpi.first.units.measure.MutDistance;
+import edu.wpi.first.units.measure.Voltage;
+import frc.robot.constants.ElevatorConstants;
+import frc.robot.utils.LoggedTunableNumber;
+import org.littletonrobotics.junction.Logger;
+
+public class ElevatorMechanism {
+    ElevatorIO io;
+    ElevatorInputsAutoLogged inputs = new ElevatorInputsAutoLogged();
+    ElevatorOutputsAutoLogged outputs = new ElevatorOutputsAutoLogged();
+
+    MutDistance goalHeight = Meters.mutable(0.0);
+    MutDistance clampedGoalHeight = Meters.mutable(0.0);
+
+    Distance minHeight = ElevatorConstants.minElevatorHeight;
+    Distance maxHeight = ElevatorConstants.maxElevatorHeight;
+
+    LoggedTunableNumber elevatorkP;
+    LoggedTunableNumber elevatorkI;
+    LoggedTunableNumber elevatorkD;
+
+    LoggedTunableNumber elevatorkS;
+    LoggedTunableNumber elevatorkV;
+    LoggedTunableNumber elevatorkA;
+    LoggedTunableNumber elevatorkG;
+
+    LoggedTunableNumber elevatorExpokV;
+    LoggedTunableNumber elevatorExpokA;
+
+    LoggedTunableNumber elevatorTuningSetpointMeters;
+    LoggedTunableNumber elevatorTuningOverrideVolts;
+
+    // Has the elevator been seeded with CRT yet?
+    // This exists in case we fail to seed with CRT the first try, it will try again each tick until
+    // it succeeds.
+    private boolean hasBeenSeeded = false;
+
+    public ElevatorMechanism(ElevatorIO io) {
+        elevatorkP =
+                new LoggedTunableNumber(
```

Comment:
I've asked them whether they want me to handle it or if they'll take care of it. After that's merged, I can create a new release on coppercore and start using maven central instead of local releases for my packages as well.
Until then, the rest of the comments are resolved so it should be ready for review again.